com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -388,14 +390,23 @@ private void Initialize(bool server)
 
             SnapshotSystem = new SnapshotSystem();
 
-            //This 'if' should never enter
             if (NetworkTickSystem != null)
             {
-                NetworkTickSystem.Dispose();
+                NetworkTickSystem.Tick -= OnNetworkManagerTick;
```

Comment:
Curious as to the "why" behind that line. Since the last reference to the tick system will be removed on the next line, what does this change?